pinot-spi/pom.xml

```diff
@@ -73,6 +73,11 @@
       <artifactId>commons-codec</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-classworlds</artifactId>
+      <version>2.8.0</version>
+    </dependency>
```

Comment:
We are starting to declare all dependencies in the dependencyManagement section in the parent pom. Given your expertise... do you recommend to do so?

Reply:
It is a matter of taste. If it is just for one dependency that is only used in 1 module, it might feel like overhead. However, if the project has this style, that's perfectly fine. At least you know where to find the version of any dependency.
I'll redeclare it.